### PR TITLE
build: setup script & default to hosted sdk

### PR DESCRIPTION
**Pull request by jatinn**
Complete each step to get your pull request merged in. [Learn more about the workflow this project uses](https://github.com/customerio/customerio-android/blob/develop/docs/dev-notes/GIT-WORKFLOW.md). 
- [ ] [Assign members of your team](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to review the pull request. 
- [ ] Wait for pull request status checks to complete. If there are problems, fix them until you see that [all status checks are passing](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fsymfony.com%2Fdoc%2F4.3%2F_images%2Fdocs-pull-request-symfonycloud.png&f=1&nofb=1). 
- [ ] Wait until the pull request has been reviewed *and approved* by a teammate
- [ ] After pull request is approved, and you determine it's ready **add the label "Ready to merge"** to the pull request and it will automatically be merged.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,8 @@ buildscript {
     ext.coroutines_version = '1.5.2'
     ext.accompanist_version = '0.23.0'
     ext.room_version = "2.4.2"
-    ext.IS_DEVELOPMENT = System.getenv("CI") == null
+    // set IS_DEVELOPMENT to true if you want to use with local version of sdk
+    ext.IS_DEVELOPMENT = false
```
Comment (levibostian) [inline build.gradle:10]: I am OK with opting in for local development when not on a CI. But the CI should always have IS_DEVELOPMENT false. 

```suggestion
    // set IS_DEVELOPMENT boolean value to true if you want to use with local version of sdk
    ext.IS_DEVELOPMENT = (System.getenv("CI") == null) && false 
```

**Review by levibostian — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Shahroz16) [reply build.gradle:10]: this please, the above check takes a lot of my brain cells 😆  

```suggestion
    // set IS_DEVELOPMENT to true if you want to use with local version of sdk
   ext.IS_DEVELOPMENT = System.getenv("CI") ? false : true
```

**Review by Shahroz16 — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jatinn) [reply build.gradle:10]: currently the local behaviour was to use the local sdk i set the value to `false` in the spirit of making it easy to run without needing the sdk and this should work for the CI as well.

developers would need to explicitly set it to `true` to use with the local version of the sdk which also involves running the local maven server from the sdk project, hence why changing the default from `true to false`.

Comment (Shahroz16) [reply build.gradle:10]: 🤦 yup you are correct, it would work for CI too, as it would always be always false and can be set true when needed.

**Pull request merged**